src/Executor.cpp

```diff
@@ -2,17 +2,19 @@
 
 #include <folly/Memory.h>
 #include <folly/futures/QueuedImmediateExecutor.h>
+#include <folly/io/IOBuf.h>
 #include "src/StackTraceUtils.h"
 #include "src/SubscriberBase.h"
 
 namespace reactivesocket {
 
 // just instantiating of the template here
 template class SubscriberBaseT<Payload>;
+template class SubscriberBaseT<folly::IOBuf>;
```

Comment:
That sounds fine, I guess. It just seems a bit inconvenient that the template will have to be explicitly instantiated for each new parameter type. Oh well.